action.yml

```
@@ -1,12 +1,21 @@
 name: "Bundler Audit Check"
 description: "Runs bundle-audit on pull requests with Gemfile.lock changes and fails the workflow if vulnerabilities are found."
 
+inputs:
+  ignore-list:
+    description: "Space-separated list of CVEs to ignore (e.g., CVE-2023-26141 CVE-2021-41182 CVE-2021-41183)"
+    required: false
+    default: ""
+
+
 runs:
   using: "composite"
   steps:
     - name: Run Bundle Audit
       uses: planningcenter/bundle-audit-action@v1
       id: bundle_audit
+      with:
+        args: ${{ inputs.ignore-list && '--ignore ' + inputs.ignore-list }}
```

I'm thinking we could also call this `ignore-list`, and then further update the `bundle-audit-action` so that it accepts `ignore-list` as an input, and call it here:

https://github.com/planningcenter/bundle-audit-action/blob/main/action.yml#L28

```
        bundle-audit check --update --ignore=${{inputs.ignore-list}} > audit_output.txt || true
```

That should work fine with an empty string as a default. And then I _think_ this line would just be

```suggestion
        ignore-list: ${{ inputs.ignore-list }}
```

---

Since I called the input `ignore_list` (snakecase) we should probably do the same here. (I'm regretting not using kebab-case but it's too late to turn back now 🙈)

```suggestion
        ignore_list: ${{ inputs.ignore_list }}
```

---

Cool, passing it into `bundle-audit-action` makes a ton of sense, fixed here: https://github.com/planningcenter/bundle-audit-check-action/pull/1/commits/2db852b5deab4715ce29a80157fe573b06f5859e